source/Nevermore/UpgradeScripts/Script0001-Nevermore-KeyAllocation.sql

```diff
@@ -1,16 +1,16 @@
-﻿IF NOT EXISTS (SELECT NULL FROM sys.tables WHERE name = 'KeyAllocation')
-	CREATE TABLE KeyAllocation
+﻿IF NOT EXISTS (SELECT NULL FROM sys.tables WHERE name = 'KeyAllocation' AND SCHEMA_NAME(schema_id) = 'dbo')
+	CREATE TABLE dbo.KeyAllocation
```

Comment:
I don't know enough about SQL schemas but is it possible that the schema was omitted on purpose?

Reply:
Server's version has `dbo`

https://github.com/OctopusDeploy/OctopusDeploy/blob/dd470d3ee864bf8b31683d2bb9b1e70b64e60ede/source/Octopus.Core/UpgradeScripts/Script0005%20-%20Key%20allocation.sql#L16

The fact that `ParameterList` has `dbo` prefixed makes me think that this was an omission